src/css/components/_multiple-choice.scss

```diff
@@ -0,0 +1,84 @@
+.cmp-multiple-choice {
+	display: flex;
+	gap: 2rem;
+	align-items: stretch;
+	margin-block: 2rem;
+	flex-wrap: wrap;
+
+	&__item {
+		flex: 1 1 calc(33% - 1.33rem);
+		position: relative;
+		display: flex;
+		align-items: stretch;
+	}
+
+	&__button {
+		position: absolute;
+		inset: 0;
+		z-index: 200;
+		background-color: transparent;
+		color: inherit;
+		border: none;
+		margin: 0;
+		padding: 0;
+		text-align: inherit;
+		font: inherit;
+		border-radius: 0;
+		appearance: none;
+		cursor: pointer;
+	}
+
+	&__answer {
+		display: flex;
+		flex-basis: 100%;
+		justify-content: center;
+		align-items: center;
+		padding: 1rem;
+		background-color: var(--background-color);
+		color: inherit;
+		font-family: inherit;
+		text-align: left;
+		font-size: clamp(1rem, 1rem + 0.5vw, 1.5rem);
+		font-weight: 400;
+		line-height: 1.2;
+		border: 0.125rem solid currentcolor;
+		border-radius: 0.25rem;
+		box-shadow: 0.25rem 0.25rem 0 0 var(--shadow);
+		transition: ease-in-out 250ms;
+		transition-property: background-color, border-color, translate, box-shadow;
+
+		> p {
+			margin: 0;
+			filter: none;
+
+			& + p {
+				margin-block-start: 0.5em;
+			}
+		}
+	}
+
+	&__button:hover + &__answer {
+		translate: 0.25rem 0.25rem;
+		box-shadow: 0 0 0 0 var(--shadow);
+	}
+
+	&__button:not([aria-pressed='true']):hover + &__answer {
+		background-color: var(--foundry-culture-accent-light);
+		border-color: var(--foundry-culture-base);
+
+		@media (prefers-color-scheme: dark) {
+			border-color: var(--brand-turquoise);
```

Comment:
The color contrast too low in dark mode, so we should update the background color here as well:

```suggestion
		background-color: var(--foundry-culture-accent-dark);
		border-color: var(--brand-turquoise);
```